recipes-bsp/lm_sensors/files/fancontrol

```diff
@@ -1,6 +1,6 @@
 # Configuration file generated by pwmconfig, changes will be lost
 INTERVAL=10
-DEVPATH=hwmon0=devices/soc0/pwm-fan hwmon1=
+DEVPATH=hwmon0=devices/platform/pwm-fan hwmon1=devices/virtual/thermal/thermal_zone0
```

Comment:
In general i'm okay with this patch, but the commit message doesn't explain why hwmon1 has been added.

Here are the relevant outputs:
Linux 4.9
root@tarragon:/sys/class/hwmon# ls -l
total 0
lrwxrwxrwx 1 root root 0 Apr 28 19:42 hwmon0 -> ../../devices/soc0/pwm-fan/hwmon/hwmon0
lrwxrwxrwx 1 root root 0 Apr 28 19:42 hwmon1 -> ../../devices/virtual/hwmon/hwmon1

Linux 6.1
root@tarragon:/sys/class/hwmon# ls -l
lrwxrwxrwx    1 root     root             0 Apr 28 19:42 hwmon0 -> ../../devices/platform/pwm-fan/hwmon/hwmon0
lrwxrwxrwx    1 root     root             0 Apr 28 19:42 hwmon1 -> ../../devices/virtual/thermal/thermal_zone0/hwmon1

Unfortunately i cannot explain why hwmon1 was empty before, except of the fact that hwmon1 is not a fan (SOC temperature sensor).